code/__defines/misc.dm

```diff
@@ -467,3 +467,9 @@ Define for getting a bitfield of adjacent turfs that meet a condition.
 #define COOK_CHECK_EXTRA	0
 #define COOK_CHECK_EXACT	1
 
+// Macros for observation events
+#define RAISE_EVENT(e, a...) decls_repository.get_decl(/decl/observ/##e).raise_event(##a)
```

Comment:
I don't think `##a` is needed here. I used it in the first argument because it acts as a literal paste.